addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -15,309 +15,431 @@ odoo.define('website_slides.fullscreen', function (require) {
     var firstScriptTag = document.getElementsByTagName('script')[0];
     firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 
+    var Header = Widget.extend({
+		events: {
+            "click .o_wslides_fs_toggle_sidebar": '_onClickToggleSidebar'
+        },
+        //--------------------------------------------------------------------------
+        // Public
+        //--------------------------------------------------------------------------
+        updateTitle: function (data) {
+            $('.o_wslides_fs_slide_title').empty().html(QWeb.render('website.course.fullscreen.title', data));
```

Comment:
This time, you should be able to use `this.$()`

Reply:
Done